src/chapters/loops/iterating-over-collections.rst

```diff
@@ -0,0 +1,86 @@
+Iterating Over Collections
+==========================
+
```

Comment:
Should we introduce the ``for (entry in string/array)`` concept in this section?

Reply:
I am ambivalent on this question. I'll add it to the agenda for our next meeting.

Reply:
OK.